pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -2552,6 +2598,14 @@ func (s *AWSCloud) EnsureLoadBalancerDeleted(service *api.Service) error {
 		}
 	}
 
+	{
+		// Remove the DNS cname if defined
+		err := s.removeLoadBalancerCnameEntry(service, lb.DNSName)
+		if err != nil {
+			glog.Warningf("Error removing cname entry for %v: %v", service.Name, err)
```

Comment:
I don't love that this is only a log warning, because it is easy to miss.  What would be the consequence of returning an error here?